cmd/cluster-init/update_sanitize_prow_jobs.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+
+	"sigs.k8s.io/yaml"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/dispatcher"
+)
+
+func updateSanitizeProwJobs(o options) error {
+	logrus.Println("Updating sanitize-prow-jobs config")
```

Comment:
nit: use the `Info/Error` etc instead of a bare print in order to allow users to choose their level of logging output

Reply:
Meant to do that already...this one was missed, good eye